### PR TITLE
Syncx

### PR DESCRIPTION
**Pull request by glennj**
None of the updated tests required any changes to the bats files.

### CODE REVIEW

File: exercises/practice/queen-attack/.docs/instructions.md [hunk 1]
```diff
@@ -8,18 +8,14 @@ A chessboard can be represented by an 8 by 8 array.
...
-2 _ _ _ _ _ B _ _ 2
-1 _ _ _ _ _ _ _ _ 1
-  a b c d e f g h
-```
+![A chess board with two queens. Arrows emanating from the queen at c5 indicate possible directions of capture along file, rank and diagonal.](https://assets.exercism.org/images/exercises/queen-attack/queen-capture.svg)
 
 You are also able to answer whether the queens can attack each other.
 In this case, that answer would be yes, they can, because both pieces share a diagonal.
+
+## Credit
```
Comment (kotp) [inline exercises/practice/queen-attack/.docs/instructions.md:16]: Should this be plural?

**Review by kotp — APPROVED** [review]
Approving with only one non-blocking question.

**Review by IsaacG — APPROVED** [review]

File: exercises/practice/queen-attack/.docs/instructions.md (hunk 1 above)
Comment (glennj) [reply exercises/practice/queen-attack/.docs/instructions.md:16]: Not necessarily: "credit where credit is due"

Comment (kotp) [reply exercises/practice/queen-attack/.docs/instructions.md:16]: That is an example wording where it iterates individually, though, as I understand the language.  The credits are shown at the end of the movie as a collection, such as we do here.

Either way, I think it is a non-blocker.

**Pull request merged**